exercises.py

```diff
@@ -1,32 +1,32 @@
 from pytest_perf.deco import extras
 
 
-@extras('perf')
-def discovery_perf():
+@extras('perf')  # type: ignore[misc]
```

Comment:
I'd like to see if it's possible to ignore this file altogether. This file has a special purpose and isn't integrated with any of the codebase. Is it possible to simply exclude it from the checks to avoid the noise?